masterlist.yaml

```diff
@@ -1200,62 +1428,97 @@ plugins:
   - name: 'CraftingSupplies_AIO_ALL.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/2359/']
     tag:
       - Invent
   - name: 'Ducks and Swans.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/2149/']
     global_priority: -90
   - name: 'BirdsOfSkyrim_SSE.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/3097/']
     after:
       - 'SkyTEST-RealisticAnimals&Predators.esp'
       - 'skyBirds_SSE.esp'
   - name: 'more idle markers.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/3211/'
+        name: 'More Tavern Idles on Nexus Mods'
     after:
       - 'Immersive Citizens - AI Overhaul.esp'
   - name: 'Hunters Not Bandits.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/1547/']
     after:
       - 'Relationship Dialogue Overhaul.esp'
   - name: 'My Home Is Your Home.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/7096/']
     global_priority: 60
     after:
       - 'AmazingFollowerTweaks.esp'
   - name: 'CFTO.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/8379/'
+        name: 'Carriage and Ferry Travel Overhaul on Nexus Mods'
     after:
       - 'Immersive Citizens - AI Overhaul.esp'
   - name: 'FlowerGirls SE.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/5941/']
     after:
       - 'Relationship Dialogue Overhaul.esp'
   - name: 'Honeyside.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/918/'
+        name: 'Sweet as Honeyside on Nexus Mods'
     after:
       - 'ELFX - NoHoneyside.esp'
       - 'Alternate Start - Live Another Life.esp'
   - name: 'Immersive Sounds - Compendium.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/523/']
     after:
       - 'Reverb and Ambiance Overhaul - Skyrim.esp'
   - name: 'Landscape Fixes For Grass Mods.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/9005/']
     after:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
   - name: 'DynDOLOD.esp'
+  # Dynamic Distant Objects LOD
+    url:
+      - link: 'https://www.nexusmods.com/skyrim/mods/59721/'
+        name: 'Dynamic Distant Objects LOD - DynDOLOD on Nexus Mods'
+      - link: 'http://forum.step-project.com/topic/13029-dyndolod-236-skyrim-se-beta-with-dynamic-lod/'
+        name: 'Dynamic Distant Objects LOD - DynDOLOD SE BETA on Step-Project Forum'
     global_priority: 115
   - name: 'ULALPatch.esp'
+    # Unique Locations - Alternate Start Patch
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/9258/'
+        name: 'Unique Locations - Riverwood Forest on Nexus Mods'
     dirty:
       - crc: 0xdde49e20
         <<: *dirtyPlugin
         itm: 4
         udr: 1
   - name: 'AyleidCitadel.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/11307/']
     tag:
       - C.Regions
   - name: '3DNPC.esp'
+  # Interesting NPCs SSE
```

Comment:
No need for the comment since you've added a named link.

Reply:
Also removed.